community/neo4j/src/test/java/upgrade/StoreUpgraderTest.java

```diff
@@ -359,6 +362,27 @@ public void upgradeShouldNotLeaveLeftoverAndMigrationDirs() throws Exception
     }
 
     @Test
+    public void upgradeShouldGiveProgressMonitorProgressMessages() throws Exception
```

Comment:
Nice test! Can we also have one for the case where we do not rebuild counts store?